packages/core/src/common/k8s-api/api-manager/api-manager.ts

```diff
@@ -41,19 +41,22 @@ export class ApiManager {
       const apis = chain(this.dependencies.apis.get().values())
         .concat(this.externalApis.values());
       const removedApis = new Set(this.apis.values());
+      const newState = new Map(this.apis);
```

Comment:
[nit] `const newApis`

Reply:
Technically it is also `oldApis` since we don't necessarily remove all of them.